src/win/fs.c

```diff
@@ -846,8 +846,13 @@ void fs__unlink(uv_fs_t* req) {
 
 void fs__mkdir(uv_fs_t* req) {
   /* TODO: use req->mode. */
-  int result = _wmkdir(req->file.pathw);
-  SET_REQ_RESULT(req, result);
+  req->result = _wmkdir(req->file.pathw);
```

Comment:
Since we are here, how about replacing `_wmkdir` with `CreateDirectoryW`? LGTM otherwise.